gyp/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
I am not a fan of these files because the hide configuration details and remove the flexibility to add different tests in different locations in the workflow.  Explicit is better that implicit so put this all in the flake8 command.

I don’t think we want to give contributors an open license to add even more complex code than we already have.  It would be more prudent to increase max-complexity to our current level and remove the C901.

Reply:
@cclauss I think this critique should be taken over to gyp-next since we're just vendoring in that code, no?